.vscode/settings.json

```diff
@@ -15,5 +15,9 @@
         "**/out": true,
         "tslint-rules/**/*.js": true
     },
-    "editor.tabSize": 2
+    "editor.tabSize": 2,
+    "prettier.semi": false,
+    "prettier.singleQuote": true,
+    "prettier.trailingComma": "es5",
+    "prettier.printWidth": 100
```

Comment:
Oh maybe I misunderstood this... so this config determines how we see the files in our editor, and then when we commit it prettifies them?